test.sh

```diff
@@ -11,6 +11,10 @@ export DOCKER=${DOCKER:-"/usr/bin/docker"}
 export SECRET=`dd if=/dev/urandom bs=4096 count=1 2> /dev/null | sha256sum`
 export ATOMIC_LIBEXEC="$(pwd)"
 
+mkdir $WORK_DIR/ostree-repo
+ostree --repo=$WORK_DIR/ostree-repo init --mode=archive-z2
```

Comment:
Shouldn't this be `--mode=bare-user` probably?  If we're using it for actual storage/checkouts.